src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolAdminOrgRoleMappingTest.java

```diff
@@ -149,185 +147,4 @@ void shouldNotReturnAdminOrgRolesForRoleId_3_to_10_with_suspendedProfile() {
-        assertThat(roleAssignments.stream().map(RoleAssignment::getRoleName).collect(Collectors.toList()),
-                containsInAnyOrder("hmcts-admin","superuser", "superuser","clerk","clerk",
-                        "task-supervisor", "task-supervisor","case-allocator", "case-allocator"));
-
-        roleAssignments.forEach(roleAssignment -> {
-            assertEquals(RoleCategory.ADMIN, roleAssignment.getRoleCategory());
-            if (!roleAssignment.getRoleName().contains("hmcts")) {
-                assertThat(new String[]{"7", "6"},
-                        ArrayMatching.hasItemInArray(roleAssignment.getAttributes().get("region").asText()));
-            } else {
-                assertNull(roleAssignment.getAttributes().get("region"));
-            }
-            if ("superuser".equals(roleAssignment.getRoleName())) {
-                assertEquals("SSCS", roleAssignment.getAttributes().get("jurisdiction").asText());
-                assertEquals("applications,hearing_work,routine_work,priority,access_requests",
-                        roleAssignment.getAttributes().get("workTypes").asText());
-            } else if ("clerk".equals(roleAssignment.getRoleName())) {
-                assertEquals("SSCS", roleAssignment.getAttributes().get("jurisdiction").asText());
-                assertEquals("applications,hearing_work,routine_work,priority",
-                        roleAssignment.getAttributes().get("workTypes").asText());
-            }
-        });
-    }
-
-    @Test
-    void shouldReturnSscsOgdOrgRolesForRoleId() {
-        allProfiles.clear();
-        Stream.of(14, 15).forEach(roleId -> {
-            CaseWorkerAccessProfile profile = TestDataBuilder.buildUserAccessProfile(roleId + "",
-                    "BBA3", false);
-            profile.setCaseAllocatorFlag("Y");
-            profile.setTaskSupervisorFlag("Y");
-            allProfiles.add(profile);
-        });
-
-        //Execute Kie session
-        List<RoleAssignment> roleAssignments =
-                buildExecuteKieSession(getFeatureFlags("sscs_wa_1_0", true));
-
-        //assertion
-        assertFalse(roleAssignments.isEmpty());
-        assertEquals(4, roleAssignments.size());
-        assertThat(roleAssignments.stream().map(RoleAssignment::getRoleName).collect(Collectors.toList()),
-                containsInAnyOrder("dwp", "hmrc","dwp", "hmrc"));
-        roleAssignments.forEach(roleAssignment -> {
-            assertEquals(RoleCategory.OTHER_GOV_DEPT, roleAssignment.getRoleCategory());
-            assertThat(new String[]{"7", "6"},
-                    ArrayMatching.hasItemInArray(roleAssignment.getAttributes().get("region").asText()));
-            assertEquals("SSCS", roleAssignment.getAttributes().get("jurisdiction").asText());
-            assertEquals("applications,hearing_work,routine_work,priority",
-                        roleAssignment.getAttributes().get("workTypes").asText());
-        });
-    }
-
```

Comment:
Removing 5 tests here and leaving the file with just 2 tests remaining, is this right?

Reply:
I removed these, as all the staff mapping have been covered more thoroughly in `DroolSscsStaffOrgRolesTest.java`

Reply:
appears these were requested to be removed by Matt, so resolved